runlog.c

```diff
@@ -312,6 +312,7 @@ run_add_record(
         int            eoln_type,
         int            variant,
         int            is_hidden,
+        int            upsolving,
```

Comment:
#32438c3

Looks like there are some other similar checks, that are not required.